enhydris_autoprocess/models.py

```diff
@@ -1,365 +0,0 @@
-import csv
-import datetime as dt
-import re
-from io import StringIO
-
-from django.db import IntegrityError, models
-from django.utils.translation import gettext_lazy as _
-
-import numpy as np
-import pandas as pd
-from haggregate import aggregate, regularize
-
-from enhydris.models import Timeseries, TimeseriesGroup
-
-from . import tasks
-
-
-class AutoProcess(models.Model):
-    timeseries_group = models.ForeignKey(TimeseriesGroup, on_delete=models.CASCADE)
-
-    class Meta:
-        verbose_name_plural = _("Auto processes")
-
-    def execute(self):
-        self.htimeseries = self._subclass.source_timeseries.get_data(
-            start_date=self._get_start_date()
-        )
-        result = self._subclass.process_timeseries()
-        self._subclass.target_timeseries.append_data(result)
-
-    @property
-    def _subclass(self):
```

Comment:
I'm confused regarding the whole `_subclass` thing, not sure what the problem we're trying to solve is.
I actually don't even know how (in the for-loop) the line `result = getattr(self, alternative)` would work? how is a variable such as `checks` assigned on the instance?

Reply:
OK, the problem with your "delete all" hack is that I can't add commits that fix the problems you find. So I created a new PR, openmeteo/enhydris-autoprocess#35. I added two commits there to address this question.